src/renderers/shaders/ShaderLib/meshphysical_frag.glsl

```diff
@@ -0,0 +1,75 @@
+#define STANDARD
```

Comment:
I wonder if this whole file could be shared with meshstandard_frag.glsl but with the addition of a few extra bits of code if PHYSICAL is defined?